crates/pop-cli/src/commands/test/contract.rs

```diff
@@ -19,15 +19,27 @@ pub(crate) struct TestContractCommand {
 impl TestContractCommand {
 	pub(crate) fn execute(&self) -> anyhow::Result<()> {
 		clear_screen()?;
+
 		if self.features.is_some() && self.features.clone().unwrap().contains("e2e-tests") {
 			intro(format!(
 				"{}: Starting end-to-end tests",
 				style(" Pop CLI ").black().on_magenta()
 			))?;
+
+			tokio::spawn(pop_telemetry::record_cli_command(
```

Comment:
Guess this might be an exception to the above comment, but think its better to have one exception that needing to touch all the above files.